package Exceptions1;

public class TestException4 {
	public static void main(String[] args) {
		int nums[] = new int[5]; // array size is 5
		System.out.println(nums[9]); // access 10 element
	}
}



